lib/ruby_llm/providers/ollama/chat.rb

```diff
@@ -12,6 +12,11 @@ def completion_url
         end
 
         def render_payload(messages, tools:, temperature:, model:, stream: false)
+          # Heuristic: if temperature is set to default and tools are in play,
+          # use a low temperature to reduce risk of small models ignoring available
+          # tools in favor of making up imaginary API calls
+          temperature = 0.1 if tools.any? && temperature == RubyLLM::Chat::DEFAULT_TEMPERATURE
```

Comment:
Another hack that touches core code.

I can cange the "if temperature is set to default" part to "if temperature is set over 0.5" to dispense with the core hack.

Reply:
Nevermind, decided against it and reverted.